### PR TITLE
RAC-836: add ge badge

### PR DESCRIPTION
**Pull request by juliensnz**
Screenshot 2021-09-01 at 11 01 01

Screenshot 2021-09-01 at 11 01 07

### CODE REVIEW

**Review by pierrejolly — APPROVED** [review]

File: src/article.handlebars [hunk 1]
```diff
@@ -9,7 +9,13 @@
             <div class="col-xs-12">
                 <p class="breadcrumb text-muted"><a href="/">Home</a> / <a href="/pim/serenity">PIM help center</a> / {{{titleWithBold}}}</p>
                 <h1 class="article-title docSearch-mainTitle">{{{titleWithBold}}}</h1>
-                {{#if eeOnly}}<p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>{{/if}}
+                {{#if eeOnly}}
+                    {{#if eeAndGeOnly}}
+                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>
+                    {{else}}
+                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a> & on Growth Edition</em></p>
```
Comment (pierrejolly) [inline src/article.handlebars:16]: 🤯
It should be the opposite

**Comment by felix-couprie** [comment]
Tested ✅:

Screen Shot 2021-09-01 at 16 01 43

Screen Shot 2021-09-01 at 15 19 40

File: src/article.handlebars [hunk 2]
```diff
@@ -9,7 +9,13 @@
             <div class="col-xs-12">
                 <p class="breadcrumb text-muted"><a href="/">Home</a> / <a href="/pim/serenity">PIM help center</a> / {{{titleWithBold}}}</p>
                 <h1 class="article-title docSearch-mainTitle">{{{titleWithBold}}}</h1>
-                {{#if eeOnly}}<p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>{{/if}}
+                {{#if eeOnly}}
+                    {{#if eeAndGeOnly}}
+                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a> & on <a href="https://www.akeneo.com/akeneo-pim-growth-edition/">Growth Edition</a></em></p>
```
Comment (felix-couprie) [inline src/article.handlebars:14]:
```suggestion
                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a> & <a href="https://www.akeneo.com/akeneo-pim-growth-edition/">Growth Edition</a></em></p>
```

**Review by felix-couprie — CHANGES_REQUESTED** [review]

File: content/md/imports-exports/tailored-export.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ themes: import-export-data
 title: Tailored export - **customize your product information exports**
 popular: false
 ee-only: true
+ge-only: true
```
Comment (felix-couprie) [inline content/md/imports-exports/tailored-export.md:7]:
```suggestion
ge-only: false
```

Comment (felix-couprie) [reply content/md/imports-exports/tailored-export.md:7]: (It's not available in GE for now)

**Review by felix-couprie — CHANGES_REQUESTED** [review]

File: src/article.handlebars [hunk 4]
```diff
@@ -9,7 +9,13 @@
             <div class="col-xs-12">
                 <p class="breadcrumb text-muted"><a href="/">Home</a> / <a href="/pim/serenity">PIM help center</a> / {{{titleWithBold}}}</p>
                 <h1 class="article-title docSearch-mainTitle">{{{titleWithBold}}}</h1>
-                {{#if eeOnly}}<p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>{{/if}}
+                {{#if eeOnly}}
+                    {{#if eeAndGeOnly}}
+                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a> & <a href="https://www.akeneo.com/akeneo-pim-growth-edition/">Growth Edition</a></em></p>
+                    {{else}}
+                        <p class="small"><em>Only available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>
```
Comment (felix-couprie) [inline src/article.handlebars:16]:
```suggestion
                        <p class="small"><em>Available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a> & <a href="https://www.akeneo.com/akeneo-pim-growth-edition/">Growth Edition</a></em></p>
                    {{else}}
                        <p class="small"><em>Available in our <a href="https://www.akeneo.com/akeneo-pim/?source=akeneo-help">Entreprise Edition</a></em></p>
```

**Review by felix-couprie — CHANGES_REQUESTED** [review]

File: src/article.handlebars (hunk 4 above)
Comment (felix-couprie) [reply src/article.handlebars:16]: Feedback from the team to get rid of the "Only" as it's not a unique choice for the first one

**Comment by felix-couprie** [comment]
Tests: ✅ OK

Screen Shot 2021-09-02 at 11 52 53

Screen Shot 2021-09-02 at 11 54 28

Screen Shot 2021-09-02 at 11 52 53

**Review by felix-couprie — APPROVED** [review]
Tests: ✅ OK

Screen Shot 2021-09-02 at 11 52 53

Screen Shot 2021-09-02 at 11 54 28

Screen Shot 2021-09-02 at 11 53 42

**Pull request merged**